web/controllers/userController/index.ts

```diff
@@ -1007,7 +1011,58 @@ async function putActive(wix_id: string, active: boolean, person: Pupil | Studen
                 await dissolveMatch(matches[i], 0, person);
             }
 
-            // Step 2: Deactivate
+            // Step 2: Cancel all courses if user is student
+            if (type == Student) {
+                let courses = await getConnection()
+                    .getRepository(Course)
+                    .createQueryBuilder("course")
+                    .leftJoinAndSelect("course.instructors", "instructors")
+                    .where("instructors.id = :id", { id: person.id})
+                    .getMany();
+
+                console.log("CRS: ", courses);
```

Comment:
Before merging this should be replaced by a proper `logger.debug` ... (just marking so that we don't forgetit)

Reply:
Yeah right, I have yet to test it.